### PR TITLE
update icon: rubymine (original, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by kilian-paquier**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *update icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)

**This PR contributes to #1125**

## Link to prove your SVG is correct and up-to-date.

https://www.jetbrains.com/fr-fr/ruby/

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Elegant

File: icons/rubymine/rubymine-plain-wordmark.svg [hunk 1]
```diff
@@ -1 +1,3 @@
...
...5-.691v-.016c0-.332.3-.582.778-.582a2.272 2.272 0 0 1 1.191.387l.27-.45a2.714 2.714 0 0 0-1.446-.445zm-13.101.012a1.734 1.734 0 0 0-1.645 1.12v-1.05h-.597v4.024h.597v-1.606c0-1.21.719-1.84 1.594-1.84h.05v-.648zm2.474 0c-.059-.002-.118 0-.177.004a3.254 3.254 0 0 0-1.461.336l.18.5a2.884 2.884 0 0 1 1.218-.297c.758 0 1.2.374 1.2 1.097v.141A4.235 4.235 0 0 0 60 74.578c-1.016 0-1.719.449-1.719 1.297v.016c0 .828.754 1.254 1.504 1.242a1.762 1.762 0 0 0 1.453-.695v.609h.575v-2.461a1.6 1.6 0 0 0-.41-1.176 1.765 1.765 0 0 0-1.124-.445zm-29.424.055 1.829 4.015c-.243.567-.466.754-.825.754a1.217 1.217 0 0 1-.629-.148l-.203.472a1.805 1.805 0 0 0 .856.203c.605 0 .996-.3 1.34-1.136l1.687-4.145h-.64l-1.29 3.34-1.464-3.355h-.66zm32.207.015v4.016h.598v-4.016h-.597zm-18.894.422c.801 0 1.22.637 1.281 1.367h-2.59c.08-.785.606-1.367 1.309-1.367zm-15.746.031c.762 0 1.418.605 1.418 1.555v.016c0 .972-.641 1.546-1.418 1.546a1.506 1.506 0 0 1-1.457-1.554v-.016a1.507 1.507 0 0 1 1.457-1.547zm22.215 1.094h1.718c.95 0 1.446.34 1.438.934v.015c0 .61-.516.977-1.34.977h-1.816v-1.926zm9.422.438a4.164 4.164 0 0 1 1.18.171v.391c0 .636-.598 1.098-1.356 1.086-.535 0-1.004-.297-1.004-.8v-.013c0-.507.427-.835 1.18-.835z"/>
```
Comment (Snailedlt) [inline icons/rubymine/rubymine-plain-wordmark.svg:2]: Missing fill?
Any reason for having this icon black while the other plain icon is orange?

Comment (kilian-paquier) [reply icons/rubymine/rubymine-plain-wordmark.svg:2]: Related to the #1291 thread

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Marking as request changes until #1291 is solved

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**